### ISSUE TITLE
A11y_Visual Studio Code Web Extensions_Github Pull request_Create Pull request_DeveloperTool: Name is not provided for combo box in 'Merge changes from' section.

### ISSUE THREAD

**Issue by sspatil123** [microsoft/vscode-pull-request-github#4237]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#GH_Visual Studio Code Web Extensions_Nov2022;#Visual Studio Code Web Extensions ;#ChromiumEdge;#DesktopWeb;#A11ySev2;#Win11;#WCAG4.1.2;#DeveloperTool;#Closed;

## Environment Details:

https://github.com/microsoft/vscode-pull-request-github/issues/4237#tasklist-block-4276087b-1852-436c-bcfe-3be728cf2954

Application Name: Visual Studio Code Web Extensions
URL: https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github
Windows Version: Win11
Edge version:107.0.1418.56

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Open https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github environment and login with git hub account.
3. tab to 'Github Pull request' and press enter.
4. tab to 'Create Pull request' and press enter.
5. Check name provide for combo box is correct or not.

## Actual:
Name is not provided for combo box in 'Merge changes from' section.

**Similar issue observed in 'INTO' Section on same screen.**

## Expected:
'Path' name should be provided for combo box in 'Merge changes from' section.

## User Impact:
Screen reader users will not get complete information of box.

## Attachment:
![NameNotProvided](https://user-images.githubusercontent.com/85863999/203697473-84ea26a0-804b-4113-8746-5796ad4fed7e.png)
![NameNotProvided_SimilarIssue](https://user-images.githubusercontent.com/85863999/203697943-4bf7cf6e-94c4-4bdc-bc23-a560b1798db2.png)

**Comment by sspatil123** [comment]
The issue is still reproing in https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github this environment
![image](https://user-images.githubusercontent.com/85863999/209779970-92781691-c44b-4936-807e-98aa815638c7.png)

**Comment by sspatil123** [comment]
The issue is fixed in environment.https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github hence closing bug
![4237_Fixed1](https://user-images.githubusercontent.com/85863999/211273786-dab76381-410d-41c1-b811-ad0a79f0096f.png)
![4237_Fixed2](https://user-images.githubusercontent.com/85863999/211273872-228057cc-f102-4abb-b75e-ef3fe97e78a8.png)
